### DISCUSSION TITLE
Strict blocking in Gmail

### DISCUSSION THREAD

**Discussion by davids74** [dhowe/AdNauseam#2444] [General]
I noticed AdNauseam kept crashing when I switch to Gmail tab.
First I disabled AdNauseam for all Google but it did not help.
I went to see if there was a crash log in the settings but found Strict Blocking tab instead where I deleted the gmail.com from the list. I wonder if it did the trick because Gmail is called mail.google.com not gmail.com
What is the reason why Gmail is added to the list? Because unlike Outlook, it does not show ads. If it is for emails, Google has pretty good defences itself. I receive significantly less spam emails in Gmail than I do in Outlook. 

I use Chromium (Legacy) Version 120.0.6077.0 (Official Build) (x86_64) and AdNauseam 3.1.18 on Mac El Capitan 10.11.5

**Comment by davids74** [comment]
I learned that unchecking Extra Privacy not only stops crashes of AdNauseam but also other plugins like github bypass-paywalls-chrome-master.

**Comment by mneunomne** [comment]
@davids74 indeed you are right about gmail. It was added to the strict block list because it's not possible to collect the ads they display, which are not the spams, but rather the fixed ads that were displayed fixed at the top of the email list. I actually don't see them anymore, so I will remove gmail from the default strict-block-list since it seems now unnecessary.
